pages.ko/common/stun.md

```diff
@@ -0,0 +1,12 @@
+# stun
+
+> 클래식 STUN 클라이언트.
+> 더 많은 정보: <https://manned.org/stun.1>.
+
+- STUN 요청 생성:
+
+`stun {{1&1 STUN 서버}}`
+
+- STUN 요청을 하고 소스 포트를 지정:
+
+`stun {{1&1 STUN 서버}} -p {{4302}}`
```

Comment:
```suggestion
`stun {{stun.1und1.de}}`

- STUN 요청을 하고 소스 포트를 지정:

`stun {{stun.1und1.de}} -p {{4302}}`
```

Thanks for your contribution, I would suggest going with the English hostname (feel free to update it to something else) as I couldn't find if it supports multilingual hostname. Alternatively, you can just translate it as "hostname" instead of giving an example.

Reply:
Hi, @kbdharun 
 
In my opinion, I am curious about the direction of setting `stun.1und1.de` as the 'STUN 호스트 네임' in Korean. 

I like your idea of ​​hostname, but what do you think about limiting the scope to STUN servers?

For that reason, I would like to suggest it because it appears to be a frequently used host name in [Public STUN server list](https://gist.github.com/mondain/b0ec1cf5f60ae726202e).

I'm curious about your thoughts. Please give me your opinion.

Thank you.

Reply:
Sure, anything that works.